.github/workflows/wheels-recipe.yaml

```diff
@@ -266,7 +266,7 @@ jobs:
       issues: write
     steps:
       - name: "Job has failed: reporting"
-        uses: JasonEtco/create-an-issue@v2
+        uses: JasonEtco/create-an-issue@1b14a70e4d8dc185e5cc76d3bec9eab20257b2c5 # v2.9.2
```

Comment:
Sorry for commenting so late after the fact. But I still think its good to check.

This action is placed after creation of the wheels and before deploy via `wheels-test-and-release.yaml`. I know the `if: ${{ ... && contains(needs.*.result, 'failure') }}` should prevent it from running for successful releases. 

But if we ever change that or there's a bug we increase our exposure during releases. Do we trust this action enough to integrate it that closely with our critical release workflow.